src/textual/keys.py

```diff
@@ -197,6 +196,18 @@ class Keys(str, Enum):
     ShiftControlEnd = ControlShiftEnd
 
 
+# We want to make sure that mypy knows that the values of Keys will always be strings.
```

Comment:
I don't know what is going on here.

Can you explain it like I'm 5?

Reply:
Fast-forward: This was the very first typing issue I tried to fix (I tackled them in alphabetical order 😅 ) and I couldn't figure out anything better.

TL;DR: the typing of Enum.value in `enum.pyi` is very meta because of the way the attribute `value` is implemented. When trying to solve this issue, I focused too much on what the typing of `value` looked like in `enum.pyi` and forgot to think about the actual implementation of `value` in `enum.py`.
Taking a look at it now inspired a simpler solution.

Reply:
It doesn't feel like the right way to do things. The fact that you have to ignore the redefinition on 205 makes me suspicious.

Think it might be better to revert this one, and we will tackle it in the future when I have time to grok it.

Reply:
Sorry, I forgot to commit and push the newest fix that looks much more reasonable.